### DISCUSSION TITLE
consensus/raft-leader-election

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#185] [General]
# consensus/raft-leader-election

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/consensus/raft-leader-election.html

**Comment by eastjoe** [comment]
"RequestVote RPC（请求投票 RPC）：Follower 在选举期间发起，用于通知其他节点投票。"
应该是Candidate发起

**Reply by isno** [reply]
谢谢提醒，你说的对，是 Candidate。
